git-hub

```diff
@@ -1431,6 +1433,61 @@ class PullCmd (IssueCmd):
 			git_quiet(1, 'fetch', remote_url, remote_branch)
 			git_quiet(1, 'checkout', 'FETCH_HEAD', *args.args)
 
+	class PushCmd (PullUtil):
+		cmd_help = "force push an update to an open pull request "
+		"and post a diff to the changes"
+
+		@classmethod
+		def setup_parser(cls, parser):
+			parser.add_argument('pull',
+				help="number identifying the pull request to update")
+			parser.add_argument('head', metavar='HEAD', nargs='?',
+				help="branch (or git ref) where your changes "
+				"are implemented")
+			parser.add_argument('-m', '--message', metavar='MSG',
+				help="text you want to add to the normal update message")
+		@classmethod
+		def run(cls, parser, args):
+			pull = req.get(cls.url(args.pull))
+
+			if pull['state'] == 'closed':
+				warnf('Updating a closed pull request '
+					'(closed at {closed_at})!', **pull)
```

Comment:
Could you abort on a closed P.R. ? IIRC last time I tried to update something closed, it screwed the P.R. up and I had to reopen it.